### PR TITLE
Mystory

### PR DESCRIPTION
**Pull request by AlexDzot**
Look at the my addition to Russian version of story.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: russian/sleep/brushing/brushing.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 Вы идете в ванную почистить зубы. В стаканчике лежит две зубные счетки,
 красная и синяя, и вы застываете в размышлении и задаете себе вопрос: 
-"Какую из зубных счеток выбрать?"
\ No newline at end of file
+"Какую из зубных счеток выбрать?"
+
+[Вы выбрали синюю щетку] (choice/blue-toothbrush.md)
+[Вы выбрали красную щетку] (choice/red-toothbrush.md)
```
Comment (casey-collab) [inline russian/sleep/brushing/brushing.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: russian/sleep/brushing/choice/blue-toothbrush.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+После того, как вы взяли в руку синюю щетку, обращается вверх тормашками и вы начинаете увязать в зефире.
```
Comment (casey-collab) [inline russian/sleep/brushing/choice/blue-toothbrush.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `russian/sleep/brushing/choice/red-toothbrush.md` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
It looks like this pull request has been replaced by [this one](https://github.com/udacity/create-your-own-adventure/pull/1242), so I'm closing it. If I've misunderstood, please re-open and let me know.

**Pull request closed without merging**